### PR TITLE
Upgrade WinUI to v2.4 and prepare for release candidate 

### PR DESCRIPTION
**Pull request by duke7553**
This PR will also improve thumbnail loading for Storage Items

EDIT 5/13/2020: We also introduce a lousy, but usable item change watching implementation. This is a bandage over the problem until future work can be done, and requires more testing in large directories.

### CODE REVIEW

File: Files/Views/Pages/Settings.xaml [hunk 1]
```diff
@@ -118,13 +119,13 @@
...
-                    Margin="0,20,0,20"
+                    Margin="0,0,0,0"
                     HorizontalAlignment="Left"
                     VerticalAlignment="Stretch"
                     Fill="{ThemeResource AppBarItemForegroundThemeBrush}"
-                    Opacity=".5"
+                    Opacity=".35"
                     Visibility="Collapsed" />
-                <Frame x:Name="SettingsContentFrame" Margin=".5,0" />
+                <Frame x:Name="SettingsContentFrame" Padding="14,0,0,0" />
```
Comment (yaichenbaum) [inline Files/Views/Pages/Settings.xaml:128]: Will this cause issues with the content getting cut off if you make the window size smaller?

File: Files/Views/Pages/Settings.xaml [hunk 2]
```diff
@@ -118,13 +119,13 @@
                 <Rectangle
                     x:Name="Divider"
                     Width=".5"
-                    Margin="0,20,0,20"
+                    Margin="0,0,0,0"
```
Comment (yaichenbaum) [inline Files/Views/Pages/Settings.xaml:122]: This margin can be removed if it is set to `0`.

File: Files/Views/Pages/ModernShellPage.xaml [hunk 3]
```diff
@@ -22,7 +22,18 @@
             <ColumnDefinition Width="*" />
         </Grid.ColumnDefinitions>
 
-        <controls:ModernSidebar x:Name="SidebarControl" Margin="0,-32,0,0" />
+        <Grid>
+            <Grid.Background>
+                <AcrylicBrush
+            Windows10version1903:TintLuminosityOpacity="0.9"
+            BackgroundSource="HostBackdrop"
+            FallbackColor="{x:Bind local1:App.AppSettings.AcrylicTheme.FallbackColor, Mode=OneWay}"
```
Comment (tsvietOK) [inline Files/Views/Pages/ModernShellPage.xaml:30]: Maybe it is better to align code?

File: Files/Views/Pages/Settings.xaml (hunk 1 above)
Comment (duke7553) [reply Files/Views/Pages/Settings.xaml:128]: Not in my testing.

Comment (yaichenbaum) [reply Files/Views/Pages/Settings.xaml:128]: Can you try resizing the window to be the smallest size before the nav menu moves to the top view. One you do that, check on every settings page that nothing gets cut off.

**Pull request merged**